sites/partners/src/listings/PaperListingForm/index.tsx

```diff
@@ -530,7 +530,7 @@ const ListingForm = ({ listing, editMode }: ListingFormProps) => {
     }
   }, [submitData.ready, submitData.data, onSubmit, status])
 
-  return (
+  return loading === true ? null : (
```

Comment:
This resolved issues I saw where the form could re-render while the form is submitting. This also appears to solve the double submit issue.